client/src/store/language.ts

```diff
@@ -1,10 +1,11 @@
 import { atom } from 'recoil';
+import Cookies from 'js-cookie';
 
 const userLang = navigator.language || navigator.languages[0];
 
 const lang = atom({
   key: 'lang',
-  default: localStorage.getItem('lang') || userLang,
+  default: Cookies.get('langcode') || userLang,
```

Comment:
I think we should keep use of localStorage as a fallback if cookies have no langcode